cli/cmd/output/task.go

```diff
@@ -0,0 +1,58 @@
+package output
+
+import (
+	"fmt"
+)
+
+const taskLineLength = 40
+
+type Task struct {
+	msg       string
+	length    int
+	lineAdded bool
+}
+
+func (t *Task) line(icon string) string {
+	ln := icon + " " + t.msg
+	for d := len(t.msg); d < t.length; d++ {
+		ln += "."
+	}
+
+	return ln
+}
+
+func (t *Task) start() {
+	ln := t.line(hourglass)
+	fmt.Print(ln)
+}
+
+func (t *Task) AddLine(prefix string, line string) {
+	if !t.lineAdded {
+		fmt.Println()
+	}
+	fmt.Println(ansiReset+ansiFaint+prefix+ansiReset, line)
+	t.lineAdded = true
+}
+
+func (t *Task) Done() {
+	ln := t.line(checkmark)
+	if !t.lineAdded {
+		fmt.Print("\r")
+	}
+	fmt.Println(ln + ansiGreen + "OK" + ansiReset)
+}
+
+func (t Task) Error() {
```

Comment:
Is there a reason why this is the only method not tied to the pointer version of task?

Reply:
Not really. I wonder if it is better practice to be consistent, or to use the pointer receiver only when modifying fields in the struct?

For now I will keep everything with a pointer receiver.